src/other/Table.js

```diff
@@ -219,23 +272,190 @@
             .remove()
         ;
         this._paginator.render();
-    };
+        
+        function fixedLabels(){
+            var divcol = context._parentElement.selectAll(".cols-wrapper").data([0]);
+            divcol
+                .enter()
+                .append("div")
+                .attr("class","cols-wrapper")
+            ;
+            divcol.selectAll(".labels-wrapper").data([0])
+                .enter()
+                .append("table")
+                .classed("labels-wrapper", true)
+            ;
+            divcol.exit().remove();
+
+            var divrow = context._parentElement.selectAll(".rows-wrapper").data([0]);
+            divrow
+                .enter()
+                .append("div")
+                .attr("class","rows-wrapper")
+            ;
+            divrow.selectAll(".labels-wrapper").data([0])
+                .enter()
+                .append("table")
+                .classed("labels-wrapper", true)
+            ;
+            divrow.exit().remove();
+
+            var rowsWrapper = context._parentElement.select(".rows-wrapper");
+            var colsWrapper = context._parentElement.select(".cols-wrapper");
+            var colLabelsWrapper = colsWrapper.select('.labels-wrapper');
+            var rowLabelsWrapper = rowsWrapper.select('.labels-wrapper');
+            var theadSelection = context.table.select('thead');
+            var colWrapperHeight = theadSelection.node().getBoundingClientRect().height;
 
+            _copyLabelContents(context._id);
+            _setOnScrollEvents(domNode);
+
+            function _setOnScrollEvents(domNode){
+                domNode.onscroll = function(e){
+                    var leftDelta = e.target.scrollLeft;
+                    var topDelta = e.target.scrollTop;
+                    colLabelsWrapper
+                        .style("margin-left", -leftDelta + "px")
+                    ;
+                    rowLabelsWrapper
+                        .style("margin-top", -topDelta + colWrapperHeight + "px")
+                    ;
+                    rowLabelsWrapper.select("thead")
+                        .style("margin-top", topDelta - colWrapperHeight + "px")
+                    ;
+                };
+            }
+            function _copyLabelContents(){
+                colLabelsWrapper.html(theadSelection.html());
+                colLabelsWrapper
+                    .style("width", context.table.style("width"))
+                ;
+                
+                var origThead = element.selectAll("th");
+                var newThead = context._parentElement.selectAll(".cols-wrapper th");
+                origThead[0].forEach(function(el, i){
```

Comment:
This should just use d3's .each() function.  Not sure why I did this.  Similar for following line